src/perf/sample/writer.cpp

```diff
@@ -188,6 +188,25 @@ bool Writer::handle(const Reader::RecordMmapType* mmap_event)
     return false;
 }
 
+bool Writer::handle(const Reader::RecordCommType* comm)
+{
+    std::string new_command{ static_cast<const char*>(comm->comm) };
+
+    Log::debug() << "Thread " << comm->tid << " in process " << comm->pid << " changed name to \""
+                 << new_command << "\"";
+
+    // update task name
+    trace_.update_thread_name(comm->tid, new_command);
```

Comment:
There is potential to improve this code by reordering and moving the `new_command`